src/antsibull_docs/collection_links.py

```diff
@@ -70,6 +70,12 @@
                 "url": "https://groups.google.com/g/ansible-project",
             }
         ],
+        "forums": [
+            {
+                "topic": "Ansible Forum",
+                "url": "https://forum.ansible.com/c/help/6/none",
```

Comment:
Is this the default link that will show if `forums` isn't explicitly set?

Reply:
No, this is only for the Communications sections of ansible.builtin (https://docs.ansible.com/ansible/devel/collections/ansible/builtin/#communication). Right now there are no default entries.